src/Parser/Druid/Balance/Modules/ResourceTracker/AstralPowerDetails.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+
+import resourceSuggest from 'Parser/Rogue/Subtlety/Modules/ResourceTracker/ResourceSuggest';
+import Analyzer from 'Parser/Core/Analyzer';
+import Tab from 'Main/Tab';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+import Icon from 'common/Icon';
+import SPELLS from 'common/SPELLS';
+import Wrapper from 'common/Wrapper';
+
+import AstralPowerTracker from './AstralPowerTracker';
+import AstralPowerBreakdown from './AstralPowerBreakdown';
+
+class AstralPowerDetails extends Analyzer {
+  static dependencies = {
+    astralPowerTracker: AstralPowerTracker,
+  };
+  
+  suggestions(when) {     
+    resourceSuggest(when, this.astralPowerTracker, {
+      spell: SPELLS.RELENTLESS_STRIKES,
```

Comment:
What's with making `SPELLS.RELENTLESS_STRIKES` the spell? Is this just because it has an icon you like?

Reply:
yeah I forgot to replace it.